api/src/pcapi/routes/backoffice/finance/finance_incidents_blueprint.py

```diff
@@ -457,6 +461,128 @@ def validate_finance_incident(finance_incident_id: int) -> utils.BackofficeRespo
     return render_finance_incident(finance_incident)
 
 
+@finance_incidents_blueprint.route("/<int:finance_incident_id>/force-debit-note", methods=["GET"])
+@utils.permission_required(perm_models.Permissions.MANAGE_INCIDENTS)
+def get_finance_incident_force_debit_note_form(finance_incident_id: int) -> utils.BackofficeResponse:
+    finance_incident = finance_models.FinanceIncident.query.filter_by(id=finance_incident_id).one_or_none()
+
+    if not finance_incident:
+        raise NotFound()
+
+    return render_template(
+        "components/turbo/modal_form.html",
+        form=empty_forms.EmptyForm(),
+        dst=url_for("backoffice_web.finance_incidents.force_debit_note", finance_incident_id=finance_incident_id),
+        div_id=f"finance-incident-force-debit-note-modal-{finance_incident_id}",
+        title="Générer une note de débit",
+        button_text="Confirmer",
+        information='Vous allez choisir le mode de compensation "Note de débit", une note de débit sera envoyée à l\'acteur culturel à la prochaine échéance. Voulez-vous continuer ?',
+    )
+
+
+@finance_incidents_blueprint.route("/<int:finance_incident_id>/force-debit-note", methods=["POST"])
+@utils.permission_required(perm_models.Permissions.MANAGE_INCIDENTS)
+def force_debit_note(finance_incident_id: int) -> utils.BackofficeResponse:
+    finance_incident = _get_incident(finance_incident_id)
+
+    if finance_incident.status != finance_models.IncidentStatus.VALIDATED:
+        flash("L'incident doit être au statut 'validé' pour effectuer cette action.", "warning")
+        return redirect(
+            request.referrer
+            or url_for("backoffice_web.finance_incidents.get_incident", finance_incident_id=finance_incident.id),
+            303,
+        )
+
+    if finance_incident.isClosed:
+        flash("Impossible d'effectuer cette action sur un incident terminé.")
+        return redirect(
+            request.referrer
+            or url_for("backoffice_web.finance_incidents.get_incident", finance_incident_id=finance_incident.id),
+            303,
+        )
```

Comment:
Suggestion (à toi de choisir) : simplifier en vérifiant une seule fois l'état. D'autant plus que cela ne devrait jamais arriver sans bidouiller, puisque le bouton n'est pas affiché dans ce cas d'échec.
Idem pour `cancel_debit_note` si tu choisis de changer.
```suggestion
    if finance_incident.status != finance_models.IncidentStatus.VALIDATED or finance_incident.isClosed:
        flash("Cette action n'est possible que pour un incident validé non terminé.", "warning")
        return redirect(
            request.referrer
            or url_for("backoffice_web.finance_incidents.get_incident", finance_incident_id=finance_incident.id),
            303,
        )
```

Reply:
C'est plus concis effectivement